Script/char.dat

```diff
@@ -802,6 +840,61 @@ guard
       "\"Not that I think about it, I was reassigned as an honor guard after that huge screw-up. I wonder if... Nah, it's probably just a coincidence.\"";
     }
   }
+
+  Config EMISSARY;
```

Comment:
I agree with you that he's insufferable, but that's the point! I wanted him to come over as a snobby, elitist, racist rich brat. Do you have any suggestions how to tweak him? I don't really want him to be less of a jerk.

All the three ambassadors were added to Attnam partly as a way to introduce more of the world of Ivan in preparation for future quests. Elianise is an elf from the Dark Forest and I would like to do a quest about elves and mistresses in the Dark Forest. Khor the orc comes from orcish freeholds and I would love to do a quest about the new orcish freeholds and dwarven fortress of Kharaz-arad, one day. Finally, the reason Lancelyn talks about the Rebellion, slaves of Aslona and golbins is that I want to make a quest concerning those - the player should be able to see both the decadent royalty of Aslona, Harvan Black-cloak the leader of the Rebellion, and also interact with goblins (both enslaved and free) more, because I find them lackluster right now, with no story or theme. So I would really like to have the emissary in Attnam, and let him be a jerk to foreshadow the villains of a different quest.